src/ranch_acceptors_sup.erl

```diff
@@ -25,28 +25,54 @@ start_link(Ref, NumAcceptors, Transport) ->
 
 init([Ref, NumAcceptors, Transport]) ->
 	TransOpts = ranch_server:get_transport_options(Ref),
-	Logger = maps:get(logger, TransOpts, error_logger),
+	NumListenSockets = maps:get(num_listen_sockets, TransOpts, 1),
 	SocketOpts = maps:get(socket_opts, TransOpts, []),
-	%% We temporarily put the logger in the process dictionary
-	%% so that it can be used from ranch:filter_options. The
-	%% interface as it currently is does not allow passing it
-	%% down otherwise.
```

Comment:
This comment shouldn't be removed, and I think you misplaced the `put(logger, Logger)/erase(logger)`, you put them around `start_listen_sockets`.

Reply:
You need to add the comment back.